### PR TITLE
Sunset the bazel run in the prow CI

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This PR is to switch the existing CI job `periodic-kubernetes-bazel-test-ppc64le` from using Bazel to `make test` as with recent changes here - https://github.com/kubernetes/kubernetes/pull/99561, now k8s does not use Bazel thing anymore.

The tool `yq` is being downloaded from https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_ppc64le for getting the value of dependent golang version for k8s from `dependencies.yaml `file.

Also modifying the cleanup job `periodic-gcs-logs-cleanup` that deletes the log folders from GCS.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/146#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/146#pullrequestreview-606887519" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/146#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `cleanup-gcs-logs-periodics.yaml` using file `config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml`
 - key `test-kubernetes-periodics.yaml` using file `config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/146):

>This PR is to switch the existing CI job `periodic-kubernetes-bazel-test-ppc64le` from using Bazel to `make test` as with recent changes here - https://github.com/kubernetes/kubernetes/pull/99561, now k8s does not use Bazel thing anymore.
>
>The tool `yq` is being downloaded from https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_ppc64le for getting the value of dependent golang version for k8s from `dependencies.yaml `file.
>
>Also modifying the cleanup job `periodic-gcs-logs-cleanup` that deletes the log folders from GCS.


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by mkumatag** [comment]
With this, I'm not sure if we need to make modification in the test grid configuration as well?!

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> With this, I'm not sure if we need to make modification in the test grid configuration as well?!

Yes we need to change https://github.com/kubernetes/test-infra/blob/master/config/testgrids/conformance/conformance-all.yaml#L342 with new gcs folder name

**Pull request merged**
